p4-16/spec/P4-16-spec.mdk

```diff
@@ -6882,11 +6921,219 @@ control ingress(inout Header_t h, inout Meta_t m,
 ~ End P4Example
 
 In this example we define a set of 7 entries, all of which invoke
-action `a_with_control_params` except for the final entry which
+action `a_params` except for the final entry which
 invokes action `a`. Once the program is loaded, these entries are
 installed in the table in the order they are enumerated in the
 program.
 
+##### Entry priorities { #sec-entry-priorities }
+
+If a table has fields where their `match_kind`s are all `exact` or
```

Comment:
I guess we have to merge https://github.com/p4lang/p4-spec/pull/1145 first

Reply:
We could merge #1145 first, but I do not see any way in which this PR depends upon changes made in #1145.  Do you?

Reply:
What each of these kinds does is described there

Reply:
I guess I would prefer if merging this PR was not blocked waiting for #1145, if #1145 is judged as not ready to merge for some reason, but this one is.  I think the text of this PR makes perfect sense even in the absence of #1145 being merged in, even if it might be more fully understood by someone first encountering what exact, lpm, and ternary mean after reading #1145's changes.  But the spec has lived for 6 years without #1145, so it doesn't seem like that should be blocking other enhancements now.

Reply:
#1145 is merged now, so resolving this comment.